### PR TITLE
Follow up for create button

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

Things included in this PR:
- Align the breadcrumbs on all the landing and detail pages (including Accounts and My Profile)
- Prevent breadcrumbs, docs, and associated actions from wrapping on smaller viewports (will have to figure a better way to handle these in the future but everything should be good until 600px for now)
- Remove `handleAddClient` from LongviewClient
- Swap out old DocumentationButton on Create Domain and Create Kubernetes

TODO:
- [x] Align the breadcrumbs on the create pages
- [x] Move create button inline with breadcrumb for Object Storage and add logic for switching between adding buckets and access keys

## Note to Reviewers
The usage of "Kubernetes" and "Kubernetes Clusters" is inconsistent. The Kubernetes Landing page is also the only instance where the breadcrumb will wrap since it's so long so I'm considering changing it to say "Kubernetes" but I want to double-check with everyone before I go and do that 💁‍♀️

### CODE REVIEW

**Review by Jskobos — COMMENTED** [review]
The docs link looks frail next to the bolded inline action on Domains landing:

Screen Shot 2020-12-14 at 3 36 51 PM

**Comment by Jskobos** [comment]
> The usage of "Kubernetes" and "Kubernetes Clusters" is inconsistent. The Kubernetes Landing page is also the only instance where the breadcrumb will wrap since it's so long so I'm considering changing it to say "Kubernetes" but I want to double-check with everyone before I go and do that 💁‍♀️

I agree, if nothing else the detail and landing text should match so there's no jump when clicking on the crumb. I'm fine with Kubernetes in both places (Kubernetes Clusters is more accurate but kind of long)

File: packages/manager/src/components/EntityHeader/EntityHeader.tsx [hunk 1]
```diff
@@ -135,15 +137,10 @@ export const EntityHeader: React.FC<HeaderProps> = props => {
             alignItems="center"
             justify="flex-end"
             wrap="nowrap"
-            xs={12}
-            sm={8}
+            xs={actions ? 9 : 4}
```
Comment (Jskobos) [inline packages/manager/src/components/EntityHeader/EntityHeader.tsx:140]: lot of magic numbers here 🤷‍♂️

**Review by Jskobos — APPROVED** [review]
Looking pretty good. None of the feedback (from earlier or here) is blocking

1. Support tickets page looks good but uses the old pattern...is it fine as is?
2. Thanks for adding the loading state to the button for creating a longview client...usually, though, buttons use the built-in loading state (w/ a spinner) included in src/components/Button.

**Review by johnwcallahan — CHANGES_REQUESTED** [review]
Couple of notes:

Domains Header wraps strangely at smaller breakpoints:
Screen Shot 2020-12-14 at 4 27 28 PM

I feel that in general the Create Buttons should have right-padding so they don't run up against the edge of the screen:

Screen Shot 2020-12-14 at 4 26 39 PM

(Also see my comment about the Object Storage buttons)

**Review by DevDW — COMMENTED** [review]
> I feel that in general the Create Buttons should have right-padding so they don't run up against the edge of the screen:

Relatedly, at tablet viewport sizes, I think the breadcrumb could use a bit of left-padding:

Screen Shot 2020-12-14 at 5 18 16 PM

Side note, looks like that table row border issue is still pestering us:

Screen Shot 2020-12-14 at 4 54 47 PM

**Review by johnwcallahan — CHANGES_REQUESTED** [review]
Left a few more comments. Additionally there's a bug on the OBJ Access Landing page where: 

1. Edit permissions of an existing Access Key. 
2. Click "Create an Access Key"
3. Observe: the drawer is in "Edit" mode.

File: packages/manager/src/components/EntityHeader/EntityHeader.tsx [hunk 2]
```diff
@@ -28,18 +28,25 @@ const useStyles = makeStyles((theme: Theme) => ({
...
+    [theme.breakpoints.down('sm')]: {
+      paddingLeft: theme.spacing(),
+      paddingRight: theme.spacing()
     }
   },
   landing: {
     marginBottom: 0,
-    paddingTop: theme.spacing(),
-    paddingBottom: theme.spacing()
+    paddingBottom: theme.spacing(),
```
[the remark this answers is not in the document]
Comment (tiffwong) [reply packages/manager/src/components/EntityHeader/EntityHeader.tsx:38]: Nope, that's a regression

**Review by johnwcallahan — APPROVED** [review]

**Pull request merged**